### PR TITLE
Add remove

### PR DESCRIPTION
**Pull request by Meltrust**
### In this milestone I:

- Served project with Webpack, as required.
- Added functionality for checkboxes on each list item.
- Added local storage functionality. 
- Separated logic in classes and modules.
- Implemented editing functionality.
- Implemented selective delete functionality.
- Added "clear completed" functionality.
- Added automatic index updating functionality upon items deletion.
- Created live demo for easy testing/reviewing https://meltrust.github.io/todix/

Thanks for the review!! :smiley:  Thanks a lot!

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -34,9 +34,9 @@ li {
   border-bottom: solid 1px rgba(194, 194, 194, 0.4);
 }
 
-.list-items-text {
-  opacity: 70%;
-}
+/* .list-items-text {
+  opacity: 75%;
+} */
```
Comment (wandji20) [inline src/style.css]: - [ ] It will be nice to remove all commented code from your code-base

**Review by wandji20 — CHANGES_REQUESTED** [review]
Hello, @Meltrust  
-----
Great :muscle:  job :raised_hands:  implementing most of the requirements for this.
However, there are a few issues you need to address to have this project approved.
## To Highlight! :clap:  :green_circle: 

:heavy_check_mark: Descriptive commit messages
:heavy_check_mark: Creating re-usable functions
:heavy_check_mark: Well crafted README file

## Status: Require Changes :recycle: :red_circle: 
Please check the comments below and under the review and make all required changes.
Optional comments are not required to mark this project approved
- [ ] It will be nice to ignore all files that are not relevant to run this project(eg .lintrc, .stylelintrc.json, .eslintrc.json, etc).

**Happy coding :computer:** 
**_Do not hesitate to make comments in this PR in case of doubts._**
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/style.css (hunk 1 above)
Comment (Emmyn5600) [reply src/style.css]: Well done

Comment (Emmyn5600) [reply src/style.css]: Well done

Comment (Emmyn5600) [reply src/style.css]: Well done

**Review by Emmyn5600 — APPROVED** [review]
Hi,

Good job so far! 👏🏼 @Meltrust 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- The live link is working properly:heavy_check_mark:
- Your project looks awesome:heavy_check_mark:
## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous reviewer.

 #### Optional suggestions
 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by Meltrust** [comment]
Thank you!

**Pull request merged**